app/models/framework_question.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class FrameworkQuestion < VersionedModel
+  validates :key, presence: true
+  validates :section, presence: true
+  validates :question_type, presence: true
```

Comment:
Should this be an enum?

Reply:
Indeed, Im just holding off until we have a full list of values before I define them which will come in following prs